src/widgets/heightandweight/heightandweight.resource.ts

```diff
@@ -15,8 +15,10 @@ function getDimensionsObservations(patientId: string) {
   return openmrsObservableFetch(
```

Comment:
If we get the expected type of this request in there we don't need to be explicit with the typing below. Also we can get rid of `[]` and `any`, which give us no guarantees.